bedrock/externalpages/templates/externalpages/pocket/includes/mobile-nav.html

```diff
@@ -0,0 +1,79 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="mobile-nav-wrapper" tabindex="-1">
+    <nav class="mobile-nav" role="dialog" aria-modal="true" aria-label="Pocket Mobile Menu">
```

Comment:
It would be good to add an `aria-expanded` attribute here, which could then be updated by the JS when the menu is opened / closed?
```suggestion
    <nav class="mobile-nav" role="dialog" aria-modal="true" aria-expanded="false" aria-label="Pocket Mobile Menu">
```